### DISCUSSION TITLE
Does the helm charts supports multiple argocd instances installation on single cluster

### DISCUSSION THREAD

**Discussion by s-u-b-h-a-k-a-r** [argoproj/argo-helm#2204] [General]
Can anyone please help me if I want to install multiple argocd instances on the same cluster and what parameters do I need to consider

**Comment by mkilchhofer** [comment]
Yes this is possible although you need to think about your detailed use case. What is your use case?
Do you want to have one cluster with "Argo CD as a service" to manage arbitrary "external" clusters?

IMO it doesn't make sense to deploy Argo CD in cluster mode then: `createClusterRoles: false`.
Also only one helm chart can install the CRDs or even better -> manage the CRDs separately: `crds.install: false`

Keep in mind that you cannot install multiple Argo CD instances within one namespace, that is not supported.

So to summarize, use a values file like this:
```yaml
createClusterRoles: false
crds:
  install: false
```
